app/assets/javascripts/expand.js

```diff
@@ -43,30 +43,27 @@ $(document).ready(function() {
         para5.setAttribute("class", "red-completion");
       }
 
-      para5.appendChild(complete_node);
+      para5.innerHTML(complete_node);
 
-      var break_line = document.createElement("br");
+      // var break_line = document.createElement("br");
 
       var title_node = document.createTextNode(title);
-      title_header.appendChild(title_node);
-      title_header.appendChild(para5);
+      title_header.innerHTML(title_node);
```

Comment:
Identifier 'title_header' is not in camel case.<br>Identifier 'title_node' is not in camel case.